crates/ruff_linter/src/rules/flake8_bugbear/rules/function_call_in_argument_default.rs

```diff
@@ -61,7 +61,7 @@ use crate::checkers::ast::Checker;
 /// ```
 ///
 /// ## Options
-/// - `flake8-bugbear.extend-immutable-calls`
+/// - `lint.flake8-bugbear.extend-immutable-calls`
```

Comment:
Did you verify manually that these still work, when rendered in the rule documentation?

Reply:
I verified some of them ;)